schema.sql

```diff
@@ -10,3 +10,28 @@ CREATE TABLE animals(
     );
     
     ALTER TABLE animals ADD species VARCHAR(50);
+
+
+    /* create table for owners' data.*/
+CREATE TABLE owners( 
```

Comment:
Please don't forget to add a `PRIMARY KEY (id)` to owners table so that we make the id row unique 📝

Reply:
Resolved